components/list/list-item-generic-layout.js

```diff
@@ -130,16 +144,25 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 	firstUpdated() {
 		this.addEventListener('keydown', this._handleKeydown.bind(this));
 		this.addEventListener('focusin', this._setFocusInfo.bind(this));
+		this.addEventListener('d2l-dropdown-open', this._onDropdownOpen.bind(this));
```

Comment:
It would also be fine to:
```
this.addEventListener('d2l-dropdown-open', () => this._dropdownOpen = true);
```
Etc.  Fine either way though.

Reply:
Good note - I'll probably do this in a different PR